app/views/characters/search.haml

```diff
@@ -37,7 +37,7 @@
                 %br
                 = check_box_tag :search_nickname, true, params[:commit] ? params[:search_nickname] : true, {class: 'vmid no-margin', style: 'margin-bottom: 3px;'}
                 = label_tag :search_nickname, "Nickname"
-                - if false # TODO
+                -# TODO
                   %br
                   = check_box_tag :search_aliases, true, params[:commit] ? params[:search_aliases] : true, {class: 'vmid no-margin', style: 'margin-bottom: 3px;'}
                   = label_tag :search_aliases, "Aliases"
```

Comment:
I think this changes the logic here - we need to comment out the indented lines below

```suggestion
                -# TODO
                  -# %br
                  -# = check_box_tag :search_aliases, true, params[:commit] ? params[:search_aliases] : true, {class: 'vmid no-margin', style: 'margin-bottom: 3px;'}
                  -# = label_tag :search_aliases, "Aliases"
```